Client/Frontend/Widgets/ThumbnailCell.swift

```diff
@@ -156,9 +156,11 @@ class ThumbnailCell: UICollectionViewCell {
             make.top.bottom.left.right.equalTo(self.imageWrapper)
         })
 
-        backgroundEffect.snp_remakeConstraints({ make in
-            make.top.bottom.left.right.equalTo(self.imageWrapper)
-        })
+        if DeviceInfo.isBlurSupported() {
```

Comment:
Could you make the lazy getter for `backgroundEffect` optional, returning `nil` if it's not supported? Then we can just use `backgroundEffect?.snp_remakeConstraints { ... }` here.